### PR TITLE
minor fix for AWS bastion host

### PR DESCRIPTION
**Pull request by yunjiang29**
a minor update for https://github.com/openshift/release/pull/28407

### CODE REVIEW

**Comment by yunjiang29** [comment]
@jianli-wei PTAL, thanks.

File: ci-operator/step-registry/aws/provision/bastionhost/aws-provision-bastionhost-commands.sh [hunk 1]
```diff
@@ -30,8 +30,8 @@ s3_bucket_name="${CLUSTER_NAME}-s3"
 bastion_ignition_file="${SHARED_DIR}/${CLUSTER_NAME}-bastion.ign"
 bastion_cf_tpl_file="${SHARED_DIR}/${CLUSTER_NAME}-bastion-cf-tpl.yaml"
 
-if [[ ! -f "${bastion_ignition_file}" ]]; then
-  echo "'${bastion_ignition_file}' not found, abort." && exit 1
+if [ "${BASTION_HOST_AMI}" == "" ] && [[ ! -f "${bastion_ignition_file}" ]]; then
```
Comment (jianli-wei) [inline ci-operator/step-registry/aws/provision/bastionhost/aws-provision-bastionhost-commands.sh:33]: Shall the conditions be of OR relationship instead?

**Review by jianli-wei — CHANGES_REQUESTED** [review]

File: ci-operator/step-registry/aws/provision/bastionhost/aws-provision-bastionhost-commands.sh (hunk 1 above)
Comment (yunjiang29) [reply ci-operator/step-registry/aws/provision/bastionhost/aws-provision-bastionhost-commands.sh:33]: Should be `AND`, the host can be provisioned if one of the following conditions is met:
1. user provides `BASTION_HOST_AMI` (not empty)
2. use dynamic host (ignition file exits)

So if both of them are not met, will abort the process.

Comment (jianli-wei) [reply ci-operator/step-registry/aws/provision/bastionhost/aws-provision-bastionhost-commands.sh:33]: Got it, anyway it looks a bit strange to me, e.g. if BASTION_HOST_AMI is specified, the ignition file would be ignored.

Comment (jianlinliu) [reply ci-operator/step-registry/aws/provision/bastionhost/aws-provision-bastionhost-commands.sh]: @yunjiang29 suggest using `[[ "${BASTION_HOST_AMI}" == "" ]]`

Comment (yunjiang29) [reply ci-operator/step-registry/aws/provision/bastionhost/aws-provision-bastionhost-commands.sh]: all comments addressed, thank you

**Pull request merged**